example/ck_tile/03_gemm/script/benchmark_mem_pipeline.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+EXE="$(find . -name tile_example_gemm_universal -type f | head -n 1)"
+VALID=0
+
+for b in "R" "C"; do
+    for m in "64" "512" "1024" "2048"; do
```

Comment:
(Here and other files) Can we define some list and reuse them?

Reply:
Fixed the comment.